server/filename.py

```diff
@@ -0,0 +1,10 @@
+# filename.py
```

Comment:
**Side Note**: In python we tend to group helper functions AKA utility functions in a file called `helpers.py` or `utils.py`. the naming convention shouldn't matter, it's up to you. But having every function written in it's separate file is not a common practice and overall will require writing more code. What do you think ?

Reply:
It makes perfect sense. And it's much easier to use.